# Plane constraints for workspace finder
# Generated by calibrate_workspace_finder.py script
normal_x: -0.018626
normal_y: -0.885234
normal_z: -0.464771
distance: 0.532530
